app/src/main/java/org/sopt/androidseminar/ReposFragment.kt

```diff
@@ -0,0 +1,68 @@
+package org.sopt.androidseminar
+
+import android.app.Fragment
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+
+class ReposFragment : Fragment() {
+    private var _binding: FragmentReposBinding? = null
+    private val binding get() =_binding ?: error("View 를 참조하기 위해 binding 이 초기화 되지 않았습니다.")
+
+    private lateinit var reposAdapter: ReposAdapter
+
+    override fun onCreateView(
+            inflater: LayoutInflater,
+            container: ViewGroup?,
+            savedInstanceState: Bundle?
+    ): View? {
+        val _binding = FragmentReposBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        reposAdapter = ReposAdapter()
+
+        binding.reposList.adapter = ReposAdapter
```

Comment:
```suggestion
        reposAdapter = ReposAdapter()

        binding.reposList.adapter = reposAdapter
```

아마 클래스 이름 들어가서 정상 작동이 안 될 수도 있을 것 같아요. 만약에 작동이 안된다고 하면 위와같이 변수를 넣어주고 해보세요. 정상적으로 리스트가 보여질겁니다.